includes/Story_Renderer/HTML.php

```diff
@@ -264,6 +267,55 @@ protected function print_analytics( $content ) {
 		return str_replace( '</amp-story>', $output . '</amp-story>', $content );
 	}
 
+	/**
+	 * Print amp-story-bookend before closing `</amp-story>`.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @param string $content String to replace.
+	 *
+	 * @return string
+	 */
+	protected function print_bookend( $content ) {
+		$share_providers = [
+			[
+				'provider' => 'facebook',
+			],
+			[
+				'provider' => 'twitter',
+			],
+			[
+				'provider' => 'linkedin',
+			],
+			[
+				'provider' => 'email',
+			],
+			[
+				'provider' => 'system',
+			],
+		];
+
+		/**
+		 * Filters the list of sharing providers in the Web Stories sharing dialog.
+		 *
+		 * @since 1.3.0
+		 *
+		 * @link https://amp.dev/documentation/components/amp-story-bookend/#social-sharing
+		 * @link https://amp.dev/documentation/components/amp-social-share/?format=stories#pre-configured-providers
+		 *
+		 * @param array[] $share_providers List of sharing providers.
+		 */
+		$share_providers = (array) apply_filters( 'web_stories_share_providers', $share_providers );
+
```

Comment:
As there is a filter here, we need to check if the `$share_provider` is an empty array. If empty, return early.

Reply:
Users my not want sharing options at all, so this is also a valid case for an empty array.

Reply:
Good call!